test/sol6/kyberStorage.js

```diff
@@ -708,6 +712,8 @@ contract('KyberStorage', function(accounts) {
                 assert(reserveData.isEntitledRebateFlag == (reserve.type != "TYPE_MOCK"), "unexpected entitled rebate flag");
             }
             Helper.assertEqualArray(await kyberStorage.getReserves(), reserveAddresses, "unexpected reserve addresses");
+            Helper.assertEqualArray(await kyberStorage.getReservesPerType(ReserveType.FPR), reserveIdsPerType[ReserveType.FPR], "unexpected reserveId");
```

Comment:
how many reserves we set in this test?

1 and 1?

if yes its not enough....

Reply:
updated